containers/grafana/Dockerfile

```diff
@@ -1,5 +1,15 @@
 # Dockerfile for scale-ci-diagnosis
 
+# Stage-1 to generate the rendered json files from jsonnet templates.
+FROM registry.access.redhat.com/ubi8/ubi-minimal as jsonnet-builder
+
+RUN microdnf install git make tar
```

Comment:
We might want to combine the RUN statements to reduce the number of layers. Thoughts?

Reply:
makes sense, updated.